internal/cmd/run/run.go

```diff
@@ -115,6 +118,10 @@ func Command() *cobra.Command {
 	sc.StringSlice("artifacts.download.match", "artifacts.download.match", []string{}, "Specifies which test artifacts to download")
 	sc.String("artifacts.download.directory", "artifacts.download.directory", "", "Specifies the location where to download test artifacts to")
 
+	// Reporters
+	sc.Bool("reporters.junit.enabled", "reporters.junit.enabled", true, "Toggle saucectl's own junit reporting on/off. This only affects the reports that saucectl itself generates as a summary of your tests. Each Job in Sauce Labs has an independent report regardless.")
```

Comment:
Just want to double check that the default to true here is intentional.

Reply:
Good catch 😅  Must have written that during post-lunch-nap-reflex (at least that's my excuse). Yeah, default should be false 👍